Makefile

```diff
@@ -373,16 +373,16 @@ DOCKER_LPT_NIMFLAGS ?= -d:chronicles_colors:none -d:insecure
 # build a docker image for the fleet
 docker-liteprotocoltester: DOCKER_LPT_TAG ?= latest
 docker-liteprotocoltester: DOCKER_LPT_NAME ?= wakuorg/liteprotocoltester:$(DOCKER_LPT_TAG)
+# --no-cache
 docker-liteprotocoltester:
 	docker build \
-	  --no-cache \
 		--build-arg="MAKE_TARGET=liteprotocoltester" \
 		--build-arg="NIMFLAGS=$(DOCKER_LPT_NIMFLAGS)" \
 		--build-arg="NIM_COMMIT=$(DOCKER_NIM_COMMIT)" \
 		--build-arg="LOG_LEVEL=TRACE" \
 		--label="commit=$(shell git rev-parse HEAD)" \
 		--label="version=$(GIT_VERSION)" \
-		--target $(TARGET) \
+		--target $(if $(filter deploy,$(DOCKER_LPT_TAG)),deployment_lpt,standalone_lpt) \
```

Comment:
what's the diff between `deployment_lpt` and `standalone_lpt`?

Reply:
It is the distinction between the different use. `deployment_lpt` stands for running in our infra environment with its own constraints. - main diff is that in deployment we need to run a supervisor ahead of tests that can control restarts.
`standalone_lpt` is the normal usage, so in lpt_runner you can easily run your tests on demand.

Reply:
Thanks for clarifying! I was wondering if we could add a comment somewhere, if not done already:)

Reply:
Yeah must add it to lpt readme at least!